tests/models/decoder_only/language/test_jamba.py

```diff
@@ -275,6 +275,44 @@ def test_state_cleanup(
                     "could be related to finished_requests_ids")
 
 
+@pytest.mark.parametrize("model", MODELS)
+@pytest.mark.parametrize("dtype", ["float"])
+def test_multistep(
+    vllm_runner,
+    model: str,
+    dtype: str,
+    example_prompts,
+) -> None:
+    # This test is verifying that multistep works correctly
+    #on mamba-like models
+    with vllm_runner(model, num_scheduler_steps=8,
```

Comment:
It would be nice to add check to make sure that the mamba cache is empty at the end of the test to catch https://github.com/vllm-project/vllm/issues/10693, but unfortunately I don't see a way to do it that's not very high effort.